### PR TITLE
fix: text missing translation

### PR DESCRIPTION
**Pull request by Zeno-sole**

### CODE REVIEW

**Comment by Zeno-sole** [comment]
Resolve: https://github.com/linuxdeepin/developer-center/issues/4384

File: translations/dde-dock_nb.ts [hunk 1]
```diff
@@ -174,7 +174,7 @@
     <name>DevCollaborationWidget</name>
     <message>
         <source>PC collaboration</source>
```
Comment (tsic404) [inline translations/dde-dock_nb.ts:176]: 其他都是亮度，这儿改错了？

**Comment by tsic404** [comment]
cmmit 5b5b11fb89ca21474df208c8494bdf3df1f8e6b8 https://github.com/linuxdeepin/dde-dock/pull/879

**Review by tsic404 — APPROVED** [review]

**Pull request merged**